authz/providers/azure/rbac/rbac.go

```diff
@@ -57,6 +58,11 @@ type AuthzInfo struct {
 	ARMEndPoint string
 }
 
+type ReviewResult struct {
```

Comment:
if this struct is intended for internal use, you can keep it private

Reply:
done